packages/matplotlib/src/html5_canvas_backend.py

```diff
@@ -38,16 +44,13 @@ class FigureCanvasHTMLCanvas(FigureCanvasWasm):
     def __init__(self, *args, **kwargs):
         FigureCanvasWasm.__init__(self, *args, **kwargs)
 
-        # A count of the fonts loaded. To support testing
-        window.font_counter = 0
-
     def create_root_element(self):
         root_element = document.createElement("div")
         document.body.appendChild(root_element)
         return root_element
 
     def get_dpi_ratio(self, context):
-        if hasattr(window, "testing"):
+        if os.environ.get("TESTING_MATPLOTLIB"):
```

Comment:
Can't we get rid of this and monkey patch `get_dpi_ratio` in the test?

Reply:
That sounds reasonable, thanks for the suggestion!